src/azure-cli/azure/cli/command_modules/network/_help.py

```diff
@@ -7044,6 +7044,36 @@
     crafted: true
 """
 
+helps['network bastion ssh'] = """
+type: command
+short-summary: SSH to a virtual machine using Tunneling from Azure Bastion.
+examples:
+  - name: SSH to virtual machine using Azure Bastion (autogenerated)
+    text: |
+        az network bastion ssh --name MyBastionHost --resource-group MyResourceGroup --target-resource-id vmResourceId --auth-type AAD/password/ssh-key-file --username xyz
+    crafted: true
+"""
+
+helps['network bastion rdp'] = """
+type: command
+short-summary: RDP to target Virtual Machine using Tunneling from Azure Bastion.
+examples:
+  - name: RDP to virtual machine using Azure Bastion (autogenerated)
+    text: |
+        az network bastion rdp --name MyBastionHost --resource-group MyResourceGroup --target-resource-id vmResourceId
+    crafted: true
+"""
+
+helps['network bastion tunnel'] = """
+type: command
+short-summary: Show a Azure bastion host machine.
+examples:
+  - name: Show a Azure bastion host machine (autogenerated)
+    text: |
+        az network bastion tunnel --name MyBastionHost --resource-group MyResourceGroup --target-resource-id vmResourceId --resource-port 111 --port 222
+    crafted: true
+"""
+
```

Comment:
Please remove `(autogenerated)` and `crafted: true` for manual examples.

```suggestion
helps['network bastion ssh'] = """
type: command
short-summary: SSH to a virtual machine using Tunneling from Azure Bastion.
examples:
  - name: SSH to virtual machine using Azure Bastion
    text: |
        az network bastion ssh --name MyBastionHost --resource-group MyResourceGroup --target-resource-id vmResourceId --auth-type AAD/password/ssh-key-file --username xyz
"""

helps['network bastion rdp'] = """
type: command
short-summary: RDP to target Virtual Machine using Tunneling from Azure Bastion.
examples:
  - name: RDP to virtual machine using Azure Bastion
    text: |
        az network bastion rdp --name MyBastionHost --resource-group MyResourceGroup --target-resource-id vmResourceId
"""

helps['network bastion tunnel'] = """
type: command
short-summary: Show a Azure bastion host machine.
examples:
  - name: Show a Azure bastion host machine
    text: |
        az network bastion tunnel --name MyBastionHost --resource-group MyResourceGroup --target-resource-id vmResourceId --resource-port 111 --port 222
"""

```

Reply:
fixed